satpy/resample.py

```diff
@@ -673,7 +675,7 @@ def mask_source_lonlats(source_def, mask):
     # the data may have additional masked pixels
     # let's compare them to see if we can use the same area
     # assume lons and lats mask are the same
-    if mask and isinstance(source_geo_def, SwathDefinition):
+    if mask is not None and isinstance(source_geo_def, SwathDefinition):
```

Comment:
I think @mraspaud merged his fix for this. You may want to rebase and take his changes which are slightly different than this.

Reply:
yes, it's fixed and merged

Reply:
Hmm, I thought I did a pull on develop, but maybe I was still on this feature branch. I'll rebase before I continue with the comments you left.